tests/run-make/clear-error-blank-output/rmake.rs

```diff
@@ -0,0 +1,16 @@
+// When an empty output file is passed to rustc, the ensuing error message
+// should be clear. However, calling file_stem on an empty path returns None,
+// which, when unwrapped, causes a panic, stopping execution of rustc
+// and printing an obscure message instead of reaching the helpful
+// error message. This test checks that the panic does not occur.
+// See https://github.com/rust-lang/rust/pull/26199
+
+use run_make_support::rustc;
+
+fn main() {
+    let output = String::from_utf8(rustc().output("").input("blank.rs").command_output().stderr)
```

Comment:
Problem: `.command_output()` should be `run()` to assert command success. Same with other uses of `.command_output()` on `rustc`. https://github.com/rust-lang/rust/pull/125752 would catch these usages.

Reply:
There are more instances of this problem in some of the `rustdoc` tests too. I will replace them too as it's fairly trivial, let me know if you think it would be wiser to keep it out of this PR.

Reply:
I think it's better if we have a follow-up PR for unrelated tests, https://github.com/rust-lang/rust/pull/125752 should fail CI if `rustdoc` has plain `command_output` invocations.